superset-frontend/src/views/CRUD/welcome/ChartTable.tsx

```diff
@@ -76,7 +85,7 @@ function ChartTable({
     t('chart'),
     addDangerToast,
     true,
-    initialFilter === 'Favorite' ? [] : mine,
+    initialFilter === 'Mine' ? mine : filteredExamples,
```

Comment:
I think typescript can help us here to avoid using plain strings for referencing these filters